### PR TITLE
Added width 100% to modal-dialog class. Fix #1365

### LINKED ISSUE #1365: [Mobile] [Chrome] UI bug, invisible buttons
__On a desktop browser there is an issue with the mobile view, when viewing on a full size screen.__

The buttons in modal dialogs have hidden text. (Perhaps this is a problem with the margin-left?)
Problem appears when the browsers window width is greater than 768px.

**Steps to reproduce:**
 - Using a desktop browser
 - Resize window to show the "Enable mobile version" & click OK
 - Resize window to show the "Enable desktop version" & click cancel
 - Try to edit any message you have sent

*Expected:*
image

*Actual:*
 image

*Expected behaviour:*
 - Buttons show in modal dialog window

*Actual behaviour:*
 - Modal dialog shows, with invisible button text
 - Buttons have on-hover effect, and can be clicked

### PR DESCRIPTION
**Pull request by yarik-vv**
Result:

Gulp test:

### CODE REVIEW

**Comment by tavurth** [comment]
Looking forward to seeing this merged

**Comment by Ryuno-Ki** [comment]
Someone runs test suite here \o/

File: app/less/app.less [hunk 1]
```diff
@@ -3933,6 +3933,8 @@ ul.chat_modal_migrate_list {
 }
 
 .modal-dialog {
+  width: 100%;
+
```
Comment (stek29) [inline app/less/app.less]: Is this empty line needed?

Comment (yarik-vv) [inline app/less/app.less]: @stek29 I think yes. In the file, this style of writing code.

**Comment by yarik-vv** [comment]
@zhukov эта проблема так и не решена. Если вы ее не хотите исправлять, то может закрыть этот pull request и issue на которое он ссылаеться, чтоб не висели в воздухе?

**Comment by tavurth** [comment]
There are pull requests here more than twice as old. 
I think the project maintainer doesn't check these boards.

**Comment by stek29** [comment]
@tavurth Project maintainer maintains more important Telegram related stuff, and doesn't always have time to check PRs like this which revert added styles.

**Comment by tavurth** [comment]
@yarik-vv Can we keep this issue open? It doesn't matter if it takes a day or a year, maybe someday the software will be open to improvements, these issues are a great roadmap.

**Comment by rodneyrod** [comment]
Could you just open this again and wait for it to be pushed?

**Comment by yarik-vv** [comment]
@tavurth @rodneyrod okay guys, sorry. Reopen in PR #1530

**Pull request closed without merging**